dotnet/Samples/ContentInsights/CloudContentInsightsService.cs

```diff
@@ -43,8 +43,8 @@ private static void CreateContentInsightsForExistingTranslations(CloudLanguageWe
                 "Resources", "Output", "input1-translated.txt")
         };
 
-        CloudGetContentInsightsForTranslationsRequest contentInsightsForTranslationsRequest =
-            new CloudGetContentInsightsForTranslationsRequest().AddTranslationId(lwClient
+        CloudContentInsightsRequest contentInsightsForTranslationsRequest =
+            new CloudContentInsightsRequest().AddTranslationId(lwClient
```

Comment:
For readability, extract lwClient.TranslateFile(translateFileRequest) to a separate variable.